transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -164,7 +164,8 @@ void remove(AbstractEpollChannel ch) throws IOException {
     @Override
     protected Queue<Runnable> newTaskQueue(int maxPendingTasks) {
         // This event loop never calls takeTask()
-        return PlatformDependent.newMpscQueue(maxPendingTasks);
+        return maxPendingTasks == Integer.MAX_VALUE ? PlatformDependent.<Runnable>newMpscQueue()
```

Comment:
we could however `MAX_VALUE` may not necessarily mean "unlimited" in practice. For these particular use cases that is the intention, but the goal was `newMpscQueue(int)` would enforce a limit while `newMpscQueue()` isn't required to enforce a limit.